webpack.common.js

```diff
@@ -113,7 +116,7 @@ module.exports = {
         'maplibre-gl/dist/maplibre-gl.css',
         'leaflet.markercluster/dist/MarkerCluster.css',
         'react-bootstrap-typeahead/css/Typeahead.css',
-        './meinberlin/apps/plans/assets/react_plans_map.jsx'
+        './meinberlin/react/projectoverview/assets/react_plans_map.jsx'
```

Comment:
not sure if this is a good idea but it never made any sense to me why the project overview map was in plans?! so i put it in a new 'app' in react folder but can easily move it back, I just have no idea why it is in plans...